src/Components/index.js

```diff
@@ -1,23 +1,88 @@
-import React, { useState } from "react";
+import React, { useEffect, useState, useContext } from "react";
 
+import { collection, deleteDoc, doc, getDocs, query ,where} from "firebase/firestore";
+
+import { fireStore, fireStorage } from "../Config/firebaseInit";
+
+import ShowContacts from "./Context/Context";
 import Header from "./Header";
 import Search from "./Search";
+import Contacts from "./Contacts";
+import NewContact from "./NewContact";
+import Pagination from "./Pagination.js";
+import { deleteObject, ref } from "firebase/storage";
 
 const Component = () => {
+    const { contacts, fillContacts } = useContext(ShowContacts);
     const [openNewContact, setStatusNewContact] = useState(false);
+    const [currentPage, setCurrentPage] = useState(1);
+    const [contactPerPage] = useState(5);
+
+    useEffect(() => getContactFromBase(),[]);
```

Comment:
please set space after semicolon, and use {}, also try to set all useEffects in the end of the methods